services/summarizer/standard/parameters.go

```diff
@@ -104,6 +108,25 @@ func WithValidatorSummaries(enabled bool) Parameter {
 	})
 }
 
+// WithvalidatorRetainPubkeys states if the module should retain balance and epoch summaries for a subset of validator.
```

Comment:
Please update comment to match the function.

Reply:
Updated !